opentracing/standard_impl.go

```diff
@@ -0,0 +1,175 @@
+package opentracing
+
+import (
+	"fmt"
+	"reflect"
+	"sync"
+	"time"
+
+	"golang.org/x/net/context"
+)
+
+// Creates and returns a standard OpenTracer which defers to `rec` and
+// `ctxIdSource` as appropriate.
+//
+// See `InitGlobalTracer()`.
+func NewStandardTracer(rec ProcessRecorder, ctxIdSource TraceContextIDSource) OpenTracer {
```

Comment:
this file is "morally equivalent" to the old version... just a straightforward implementation of the interface.